infra/base-images/base-runner/test_all

```diff
@@ -41,7 +41,7 @@ for FUZZER_BINARY in $(find $OUT/ -executable -type f); do
     export AFL_NO_UI=1
     timeout --preserve-status -s INT 20s run_fuzzer $FUZZER
   fi
-  bad_build_check $FUZZER_BINARY
+
```

Comment:
to prevent all this code duplication, could we just wrap lines 36-43 with another if? we can set an env var (e.g. SKIP_TEST_TARGET_RUN) that controls this in the container builder step.

Reply:
Using env var to control that SGTM! Will upload an update, thanks!